lib/clipboard_image_paste/attachment_patch.rb

```diff
@@ -13,34 +13,17 @@
 #*******************************************************************************
 
 module AttachmentPatch
-  def self.included(base)
-    base.send(:include, InstanceMethods)
-
-    # Same as typing in the class
-    base.class_eval do
-      # Send unloadable so it will not be unloaded in development
-      unloadable
```

Comment:
We can remove this because we call `require_dependency` in `to_prepare` block.

Reply:
You're right unloadable breaks auto-reloading.